src/llmq/quorums_instantsend.cpp

```diff
@@ -843,7 +843,11 @@ void CInstantSendManager::RetryLockTxs(const uint256& lockedParentTx)
         }
     }
 
-    while (pindexWalk && pindexWalk != pindexLastChainLockBlock) {
+    // scan blocks until we hit the last chainlocked block we know of. Also stop scanning after a depth of 6 to avoid
+    // signing thousands of TXs at once. Also, after a depth of 6, blocks get eligible for ChainLocking even if unsafe
+    // TXs are included, so there is no need to retroactively sign these.
+    int depth = 6;
```

Comment:
```suggestion
    int depth = 0;
```

Reply:
Whoops :see_no_evil: This one was silly...and I'm actually trying to figure out since 10 minutes what's wrong with tests :D